packages/gitmoji-changelog-core/src/utils.js

```diff
@@ -0,0 +1,32 @@
+const levenshtein = require('fast-levenshtein')
+
+function groupSentencesByDistance(texts = []) {
+  const textsWithSortedWords = texts.map(text => text.split(' ').sort().join(''))
+
+  const alreadyProcessedWords = new Set()
+  const keyGroups = new Map()
+
+  for (let i = 0; i < textsWithSortedWords.length; i += 1) {
+    if (!alreadyProcessedWords.has(i)) {
```

Comment:
Maybe we should reverse the condition `if (alreadyProcessedWords.has(i)) continue` so we reduce the indentation

Reply:
+ I think linter cry about it